control-plane/agents/Cargo.toml

```diff
@@ -22,46 +21,47 @@ name = "jsongrpc"
 path = "src/bin/jsongrpc/main.rs"
 
 [[bin]]
- name = "agent-ha-node"
- path = "src/bin/ha/node/main.rs"
+name = "agent-ha-node"
+path = "src/bin/ha/node/main.rs"
 
 [[bin]]
- name = "agent-ha-cluster"
- path = "src/bin/ha/cluster/main.rs"
+name = "agent-ha-cluster"
+path = "src/bin/ha/cluster/main.rs"
 
 [dependencies]
-anyhow = "1.0.75"
-uuid = { version = "1.4.1", features = ["serde", "v4"] }
-chrono = "0.4.31"
-clap = { version = "4.4.6", features = ["color", "derive", "env", "string"] }
-tokio = { version = "1.32.0", features = ["full"] }
-tonic = "0.10.2"
-futures = "0.3.28"
-serde_json = "1.0.107"
-async-trait = "0.1.73"
+anyhow = "1.0.92"
+uuid = { version = "1.11.0", features = ["serde", "v4"] }
+chrono = "0.4.38"
+clap = { version = "4.5.20", features = ["color", "derive", "env", "string"] }
+tokio = { version = "1.41.0", features = ["full"] }
+tonic = "0.12.3"
+futures = "0.3.31"
+serde_json = "1.0.132"
+async-trait = "0.1.83"
 dyn-clonable = "0.9.0"
-snafu = "0.7.5"
+snafu = "0.8.5"
 humantime = "2.1.0"
 state = "0.6.0"
-http = "0.2.9"
-reqwest = "0.11.22"
-parking_lot = "0.12.1"
-itertools = "0.11.0"
-once_cell = "1.18.0"
-indexmap = "2.0.2"
-futures-util = { version = "0.3.28" }
-crossbeam-queue = "0.3.8"
-tower = { version = "0.4.13", features = [ "timeout", "util" ] }
-hyper = { version = "0.14.27", features = [ "client", "http1", "http2", "tcp", "stream" ] }
-opentelemetry = { version = "0.22.0" }
-tracing = "0.1.37"
-nix = { version = "0.27.1", default-features = false }
-prost-types = "0.12.1"
-url = "2.4.1"
+http = "1.1.0"
```

Comment:
nit: sort in alphabetical order? :)

Reply:
Moved it, but not everything is alphabetical, I'll leave that for another PR